### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by flutistar**
*Issue #:* /bcgov/entity#24127

*Description of changes:*
- Updated file upload component
- Created generic error handling modal
- Created `BcrosDocFetch` composoable for error reporting
- Small static text updates

**File upload component**

image

**Error modal**

image

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-88-19980c3u.web.app

File: document-service/documents-ui/src/app.vue [hunk 1]
```diff
@@ -11,7 +11,8 @@ const { isLoading } = storeToRefs(useBcrosDocuments())
           class="animate-spin text-[50px] text-blue-500 absolute top-40 left-[50%]"
         />
       </template>
-
       <NuxtPage />
+      <ModalError />
+      <UNotifications />
```
Comment (cameron-eyds) [inline document-service/documents-ui/src/app.vue:16]: I love the architecture of Nuxt, easy on the root layer for global components.

**Review by cameron-eyds — APPROVED** [review]
Looks great sir
Nice architecture, good comments and unit testing. 
Well done!

**Comment by flutistar** [comment]
Thanks @cameron-eyds

**Pull request merged**